tests/VirtoCommerce.CustomerReviews.Test/CustomerReviewsTests.cs

```diff
@@ -53,7 +52,7 @@ public async Task CanDoCRUDandSearch()
 
             // Read non-existing item
 
-            result = await CustomerReviewService().GetByIdsAsync(new[] { CustomerReviewId });
+            result = await CustomerReviewService().GetAsync(new List<string>() { CustomerReviewId });

```

Comment:
Use ICrudService<>.GetByIdAsync() instead

Reply:
Original tests work with the collection of reviews in results. I just fix the obsolete warning.